src/main/java/uk/gov/ons/census/casesvc/service/QuestionnaireLinkedService.java

```diff
@@ -59,4 +61,12 @@ public void processQuestionnaireLinked(ResponseManagementEvent questionnaireLink
         questionnaireLinkedEvent.getEvent(),
         convertObjectToJson(uac));
   }
+
+  private void checkQidNotLinkedToAnotherCase(UacDTO uac, UacQidLink uacQidLink) {
+    if (uacQidLink.getCaze() != null) {
+      if (!uacQidLink.getCaze().getCaseId().equals(UUID.fromString(uac.getCaseId()))) {
```

Comment:
what's wrong with an `&&` and combining these two conditions into a single if statement?